tests/unit/Formatters/GeoCoordinateFormatterTest.php

```diff
@@ -401,6 +401,21 @@ public function decimalMinuteSecondNotationProvider() {
 				'0.1',
 				'-0° 6\', 0° 6\''
 			),
+			'Bug T150085 with 1 second precision' => array(
```

Comment:
Would be cool if this could be described without referencing an issue. Not sure if this can be done nicely tho